### PR TITLE
Bug 1876701: Fix some spec descriptor widgets inline edit behavior

### PR DESCRIPTION
**Pull request by TheRealJon**
Fix incorrectly formatted patch path in configureSizeModal and configureUpdateStrategyModal.

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-size.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const configureSizeModal = ({
     defaultValue: specValue || 0,
     title: `Modify ${specDescriptor.displayName}`,
     message: specDescriptor.description,
-    path: `/spec/${specDescriptor.path}`,
+    path: `/spec/${specDescriptor.path.replace(/\]/g, '').replace(/[.[]/g, '/')}`,
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-size.tsx:17]: I'd suggest using `_.toPath` and then building up the patch path from the array. Something like (untested)

```suggestion
    path: ['spec', _.toPath(specDescriptor.path)].join('/'),
```

**Comment by spadgett** [comment]
Are the other descriptors that could have this problem?

File: frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-size.tsx (hunk 1 above)
Comment (TheRealJon) [reply frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-size.tsx:17]: @spadgett fixed

**Comment by TheRealJon** [comment]
Yes, already found another one. Let me scour through them and fix them all.

**Comment by TheRealJon** [comment]
@spadgett Turns out there was only one other instance of this problem in the descriptors. Should be fixed now.

**Review by spadgett — APPROVED** [review]
/lgtm

👍

**Pull request merged**